script/bump-version.py

```diff
@@ -1,61 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Safe to remove - bump logic handled in Jenkins - see https://github.com/brave/devops/blob/c6867ebc0740b6c74cb642084565213cee1b3399/jenkins/jobs/browser/brave-browser-build.yml#L121-L185